akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -550,7 +550,9 @@ class InterpreterSpec extends AkkaSpec with GraphInterpreterSpecKit {
       new InvalidAbsorbTermination)) {
       lastEvents() should be(Set.empty)
 
-      EventFilter[UnsupportedOperationException]("It is not allowed to call absorbTermination() from onDownstreamFinish.", occurrences = 1).intercept {
+      // TODO we get two log entries after more verbose error logging was added to the interpreter
+      // couldn't figure out how to not do that without breaking other expected error logging
+      EventFilter[UnsupportedOperationException]("It is not allowed to call absorbTermination() from onDownstreamFinish.", occurrences = 2).intercept {
```

Comment:
It's this one in addition to the logging I added: https://github.com/akka/akka/blob/wip-19469-better-stream-error-logging-johanandren/akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala#L715 but if I remove this then tests failed because not all code paths there goes through the error logging I added here https://github.com/akka/akka/blob/wip-19469-better-stream-error-logging-johanandren/akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala#L540